src/initializers/i18n.ts

```diff
@@ -49,4 +49,27 @@ export const setUpI18n = async (): Promise<void> => {
     })
 
   moment.locale(i18n.language)
+  setHTMLDirection(shortenLanguageCode(i18n.language))
+}
+
+export const setHTMLDirection = (language: string): void => {
+  console.log(language)
```

Comment:
Why do we output/log this?

Reply:
that's 'debug' code

Reply:
Yes, and debug-code should not be committed, should it?

Reply:
dont use console.log. Use console.debug please.